knees4bees/rancid-tomatillos
# Feature/display movies

## Is this a fix, feature, or refactor?
Feature

## What is the change?
Adds basic component structure and minimal styling to landing page.

## What does it fix?
N/A

## Where should the reviewer start?
Review all new files.

## How should this be tested?
Run the app in the browser and confirm that all elements are present: nav items, hero image and words, and individual movie tiles.

## Relevant ScreenShots
N/A
